src/main.ts

```diff
@@ -53,6 +53,10 @@ class Honk {
     if (random === 69) {
       this.client.createMessage(message.channel.id, "nice");
     }
+    const random2 = Math.floor(Math.random() * 1000);
+    if (random2 === 420 && message?.member && client.getRESTGuildMember(message.guildID, client.user.id).?permissions?.has(BigInt(134217728)) {
```

Comment:
the client member is guarenteed to be populated even without the member intent, you can simply use `message.guild.members.get(client.user.id)`

also you have a `.?` instead of `?.`

Reply:
> also you have a .? instead of ?.
fack, i typed this in the github editor :Waaa:
> message.guild.members.get(client.user.id)
I was trying to avoid using .guild.members.get lols